Tasks/Common/packaging-common/provenance.ts

```diff
@@ -51,11 +51,15 @@ export class ProvenanceHelper {
         baseUrl: string,
         handlers: VsoBaseInterfaces.IRequestHandler[],
         options: VsoBaseInterfaces.IRequestOptions): Promise<string> {
-        
-        // while this feature is in preview, it is enabled by the following variable
-        const useSessionEnabled = tl.getVariable("Packaging.SavePublishMetadata");
-        if (useSessionEnabled) {
-            tl.debug("Creating provenance session");
+               
+        const publishPackageMetadata = tl.getInput("publishPackageMetadata");
```

Comment:
What exactly is the difference between these two variables? It looks like both are doing the same thing.

Reply:
One is a variable, one is an input. Since the input is true by default, the variable is basically a heavy-handed way to disable this behaviour without modifying the build pipeline.

Reply:
cool, LGTM